doc/source/symbol-tables.rst

```diff
@@ -38,7 +38,7 @@ following command:
 The :envvar:`PYTHONPATH` environment variable is not required if the Volatility library is installed in the system's library path
 or a virtual environment.
 
-Mac/Linux symbol tables
+Mac or Linux symbol tables
```

Comment:
Is there any particular reason to change this?

Reply:
> Is there any particular reason to change this?

Yes it's because I was not able to cross reference the heading when it had slash.